### DISCUSSION TITLE
How to handle uncovered lines that are from tracing

### DISCUSSION THREAD

**Discussion by juliusl** [xd009642/tarpaulin#1538] [Q&A]
Trying out tarpaulin for the first time. I noticed that some of the uncovered lines from the coverage results are for lines of tracing such as, 

```rs
trace!("trace message");
```

I tried setting `RUST_LOG=` naively to see if that would just work, but it doesn't seem to change the results. Is there a recommend way of handling this kind of case?

**Comment by xd009642** [comment]
So in your test are you setting up a tracing subscriber? Because if not I doubt that will work. 

One approach that may work would be adding in [tracing-test](https://docs.rs/tracing-test/latest/tracing_test/) that sets up a subscriber per test and may solve things.

**Reply by juliusl** [reply]
I see, so I enable tracing-test on some tests but not all. I'll see if that changes the summary.

**Reply by juliusl** [reply]
Doesn't seem to change the coverage output even though I'm seeing the events being emitted. Because it's a macro, is it possible there just happens to be an inert block when the macro gets expanded?
